lib/improver/ensemble_calibration/ensemble_calibration.py

```diff
@@ -1018,22 +910,26 @@ def format_calibration_method(calibration_method):
                  "nonhomogeneous gaussian regression"]):
             if (format_calibration_method(self.distribution) in
                     ["gaussian", "truncated gaussian"]):
+                current_cycle = datetime_to_cycletime(
```

Comment:
Should you pass in the current cycle? If not it should be taken from the "forecast_reference_time" coordinate rather than "time"?

Reply:
You're correct that this should be the `forecast_reference_time` coordinate. I could pass in the `current_cycle` to the ensemble_calibration class, however, in some ways this is duplicating information that is already available.  What do you think?

Reply:
Yes that's fine, I don't think we need to pass it in.